PyRDF/__init__.py

```diff
@@ -55,3 +56,25 @@ def include(includes_list):
         includes_list = [includes_list]
 
     includes.extend(includes_list)
+
+def initialize(fun, *args, **kwargs):
+    """
+    Set a function that will be executed as a first step on every backend before
+    any other operation.
+
+    This allows users to inject and execute custom code on the worker environment
+    without being part of the RDataFrame computational graph.
+
+    Parameters
+    ----------
+    fun : function
+        Function to be executed.
+
+    *args
+        Variable length argument list used to execute the function.
+
+    **kwargs
+        Keyword arguments used to execute the function.
+
+    """
+    Backend.register_initialization(fun, *args, **kwargs)
```

Comment:
`initialize` may have some limitations, since not every function can be pickled. If the function passed by the user is not a pure function, then pickle will complain at runtime, making it harder for the user to understand the issue. We can add a couple of checks prior the call to `register_initialization` where we try to pickle the user function and continue only if it succeeds.

Reply:
@etejedor What do you think about this?

Reply:
Indeed, this might be part of a more general series of checks in the local machine before going to a distributed environment (added as an issue #12).